### PR TITLE
feat(ios): Added support for showing splash screen over active view on IOS. App can contain several views

### PR DESCRIPTION
**Pull request by alexyazvinsky**
…can contain several views.

### CODE REVIEW

File: src/ios/CDVSplashScreen.m [hunk 1]
```diff
@@ -60,6 +60,30 @@ - (void)observeValueForKeyPath:(NSString*)keyPath ofObject:(id)object change:(NS
     [self updateImage];
 }
 
+- (UIViewController*)getCurrentViewController {
+    // Grab the view controller that is currently presented.
+    UIViewController *currentViewController = [[[UIApplication sharedApplication] delegate] window].rootViewController;
+    UIViewController *keyRootController = [UIApplication sharedApplication].keyWindow.rootViewController;
+    
+    // Now present the alert on the view controller that is currently presenting.
```
Comment (janpio) [inline src/ios/CDVSplashScreen.m]: What does "alert" refer to here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "cordova-plugin-splashscreen",
-  "version": "5.0.4-dev",
+  "name": "dc-cordova-plugin-splashscreen",
```
Comment (janpio) [inline package.json]: This change (and the version) has to be removed if this should be merged to the original Cordova plugin.

File: src/ios/CDVSplashScreen.m (hunk 1 above)
Comment (alexyazvinsky) [reply src/ios/CDVSplashScreen.m]: Comment has wrong text, I will remove it.

File: package.json (hunk 2 above)
Comment (alexyazvinsky) [reply package.json]: I will remove it.

**Comment by alexyazvinsky** [comment]
Fixed.

**Comment by janpio** [comment]
CI, rerun the tests please.

**Comment by erisu** [comment]
The SplashScreen plugin for iOS has been integrated into the `cordova-ios` platform repo as a core feature of iOS. The iOS source code no longer resides in this repo and has been removed.

If you believe that this is a still needed feature, please create a new PR with the change in the `cordova-ios` repo.

**Pull request closed without merging**